### PR TITLE
Merging bug fixes made to CITZ PIMS

### PR DESCRIPTION
**Pull request by Fosol**
There were a number of bug fixes related to notifications, templates, and the Common Hosted Email Service.  The database migrations are not important to MOTI, but also won't hurt anything as they are only changes to notifications related to Disposal Projects.

Agency Interest now has three options instead of two, Interested, Not Interested, and Send Notifications.  Both Interested and Not Interested cancels any pending notifications.

- Fix notifications
- Fix notification templates
- Add database migration
- Fix disposal projects in ERP
- Fix agency interest response
- Fix project approval requests
- Add new agency response type
- Fix CHES cancel response
- Add request logging
- Add unit tests

### CODE REVIEW

File: backend/dal/Migrations/Initial/Up/PostUp/01-NotificationTemplates.sql [hunk 1]
```diff
@@ -105,7 +105,7 @@ INSERT INTO dbo.[NotificationTemplates] (
     , 'Inform owning agency their disposal project was denied.'
     , 0
     , ''
-    , 'Deibert, Yvonne CITZ:EX <Yvonne.Deibert@gov.bc.ca>; Uppal, Sanjay CITZ:EX <Sanjay.Uppal@gov.bc.ca>; Miller, Lauren CITZ:EX <Lauren.Miller@gov.bc.ca>'
+    , ''
```
Comment (Fosol) [inline backend/dal/Migrations/Initial/Up/PostUp/01-NotificationTemplates.sql:108]: Removing email addresses from repo

File: backend/dal/Migrations/v01.09.02/Up/PostUp/01-NotificationTemplates.sql [hunk 2]
```diff
@@ -1,4 +1,4 @@
 PRINT 'Updating NotificationTemplates - Remove CC'
 
 UPDATE dbo.[NotificationTemplates]
-SET [Cc] = REPLACE([Cc], 'Deibert, Yvonne CITZ:EX <Yvonne.Deibert@gov.bc.ca>; Uppal, Sanjay CITZ:EX <Sanjay.Uppal@gov.bc.ca>; Miller, Lauren CITZ:EX <Lauren.Miller@gov.bc.ca>', '')
```
Comment (Fosol) [inline backend/dal/Migrations/v01.09.02/Up/PostUp/01-NotificationTemplates.sql:4]: Removing email addresses from repo

File: frontend/src/components/common/form/ParentSelect.tsx [hunk 3]
```diff
@@ -100,9 +100,9 @@ export const ParentSelect: React.FC<IParentSelect> = ({
     if (value?.value) {
       return [value];
     }
-    if (value !== undefined && !_.isEmpty(value.toString())) {
+    if (value !== undefined && !_.isEmpty(value?.toString())) {
```
Comment (Fosol) [inline frontend/src/components/common/form/ParentSelect.tsx:103]: Fix bug

**Review by jdtoombs — APPROVED** [review]

File: frontend/src/features/projects/list/ProjectApprovalRequestListView.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { ProjectListView } from '.';
+
+/**
+ * A list view paging table to search for projects that have been submitted for assessment.
+ * @returns ProjectApprovalRequestListView component.
+ */
+export const ProjectApprovalRequestListView = () => {
+  console.info('test');
```
Comment (devinleighsmith) [inline frontend/src/features/projects/list/ProjectApprovalRequestListView.tsx:10]: this should probably be removed.

Comment (devinleighsmith) [reply frontend/src/features/projects/list/ProjectApprovalRequestListView.tsx:10]: also is this a bug fix? this seems like new functionality.

**Review by devinleighsmith — APPROVED** [review]

File: frontend/src/features/projects/list/ProjectApprovalRequestListView.tsx (hunk 4 above)
Comment (Fosol) [reply frontend/src/features/projects/list/ProjectApprovalRequestListView.tsx:10]: This was to fix bugs within CITZ PIMS.

**Pull request merged**